pkg/cable/wireguard/driver.go

```diff
@@ -5,22 +5,62 @@ import (
 	"fmt"
 	"net"
 	"os"
+	"strconv"
 	"sync"
 	"time"
 
 	"github.com/kelseyhightower/envconfig"
-
-	v1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/vishvananda/netlink"
+	"golang.zx2c4.com/wireguard/wgctrl"
+	"golang.zx2c4.com/wireguard/wgctrl/wgtypes"
+	"k8s.io/klog"
 
 	"github.com/submariner-io/admiral/pkg/log"
+	v1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
 	"github.com/submariner-io/submariner/pkg/cable"
 	"github.com/submariner-io/submariner/pkg/types"
-	"github.com/vishvananda/netlink"
+)
 
-	"k8s.io/klog"
+var wireguardConnectedEndpoints = prometheus.NewGauge(prometheus.GaugeOpts{
+	Name: "wireguard_connected_endpoints",
+	Help: "wireguard connected endpoints",
+})
+
+var endpointLabels = []string{
+	// destination clusterID
+	"dst_clusterID",
+	// destination Endpoint hostname
+	"dst_EndPoint_hostname",
```

Comment:
ignorant question, is it common to use labels like this in prometheus? or something like dst_endpoint_hostname ?

Reply:
https://prometheus.io/docs/practices/naming/#labels

Reply:
I don't think I understood your question. 
In any case - I renamed the metrics.